docs/user/accumulo/install.rst

```diff
@@ -84,49 +65,57 @@ Namespace Install
-
-.. code::
-
-    $ ./setup-namespace.sh -u myUser -n myNamespace
-
-The command line arguments the script accepts are:
-
-* -u <Accumulo username>
-* -n <Accumulo namespace>
-* -p <Accumulo password>
-* -t <Use a cached Kerberos TGT>
-* -g <Path of GeoMesa distributed runtime JAR> (optional, will default to the distribution folder)
-* -h <HDFS URI e.g. hdfs://localhost:9000> (optional, will attempt to determine if not supplied)
+To install the distributed runtime JAR, first copy it into HDFS:
 
-Since ``accumulo shell`` does not directly support Kerberos keytabs, if using Kerberos (``-t``) then a cached Kerberos
-ticket-granting-ticket (TGT) should be obtained using the ``kinit`` command.
+.. code-block:: bash
 
-If ``-t`` is specified, ``-p`` must not be specified. If both ``-p`` and ``-t`` are omitted, the user is prompted for a password.
+    $ hadoop fs -mkdir /accumulo/classpath/myNamespace
+    $ hadoop fs -put \
+      geomesa-accumulo_2.11-$VERSION/dist/accumulo/geomesa-accumulo-distributed-runtime_2.11-$VERSION.jar \
+      /accumulo/classpath/myNamespace/
 
-Alternatively you can manually install the distributed runtime JAR with these commands:
+Then configure the namespace classpath through the Accumulo shell:
 
-.. code::
+.. code-block:: bash
 
     $ accumulo shell -u root
     > createnamespace myNamespace
     > grant NameSpace.CREATE_TABLE -ns myNamespace -u myUser
-    > config -s general.vfs.context.classpath.myNamespace=hdfs://NAME_NODE_FDQN:54310/accumulo/classpath/myNamespace/[^.].*.jar
+    > config -s general.vfs.context.classpath.myNamespace=hdfs://NAME_NODE_FDQN:9000/accumulo/classpath/myNamespace/[^.].*.jar
     > config -ns myNamespace -s table.classpath.context=myNamespace
 
-Then copy the distributed runtime JAR into HDFS under the path you specified.
-The path above is just an example; you can included nested folders with project
-names, version numbers, and other information in order to have different versions of GeoMesa on
-the same Accumulo instance. You should remove any GeoMesa JARs under
-``$ACCUMULO_HOME/lib/ext`` to prevent any classpath conflicts.
+.. note::
+
+    The path above is just an example; you can included nested folders with project
+    names, version numbers, and other information in order to have different versions of GeoMesa on
+    the same Accumulo instance. You should remove any GeoMesa JARs under
+    ``$ACCUMULO_HOME/lib/ext`` to prevent classpath conflicts.
 
 .. note::
 
     When connecting to a data store using Accumulo namespaces, you must prefix
-    the ``tableName`` parameter with the namespace. For example, refer to the
+    the ``accumulo.catalog`` parameter with the namespace. For example, refer to the
     ``my_catalog`` table as ``myNamespace.my_catalog``.
 
+.. _install_accumulo_runtime_manual:
+
+System Install
+^^^^^^^^^^^^^^
+
+Alternatively, the GeoMesa runtime JAR can be copied into the ``$ACCUMULO_HOME/lib/ext`` folder on
```

Comment:
Ah, nevermind the above comment.  Is lib/ext still a thing in Accumulo 2.0?

Reply:
i believe so... i didn't actually verify it though, and it doesn't exist by default.

Reply:
actually I think @khobbs-ccri has been using lib/ext with accumulo 2

Reply:
No, I've been using the namespace install. I think lib ext went away and, i think, creating it didn't work.

Reply:
Ok, given that, should this documentation be updated?  (If so, can we do it in this PR?  Either way, who's the right person to spill some words on that?)

Reply:
accumulo still mentions `lib/ext` in their docs: https://accumulo.apache.org/docs/2.x/getting-started/table_configuration#iterators

Reply:
The namespace install worked perfectly. but I really didn't explore my one failed attempt at using lib/ext further.